builder/vsphere/supervisor/step_import_image_test.go

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package supervisor_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	"github.com/vmware-tanzu/image-registry-operator-api/api/v1alpha1"
```

Comment:
I think it would be better to alias this so that les changes are needed in the future when this is past alpha.

Reply:
Yep, updated to imgregv1